File: sbe/binance/spot_3_2/generated-php/spot_sbe/OrderStatus.php
<?php
/**
 * Generated SBE (Simple Binary Encoding) message codec.
 */

class OrderStatus
{
    public const NEW = 0;
    public const PARTIALLYFILLED = 1;
    public const FILLED = 2;
    public const CANCELED = 3;
    public const PENDINGCANCEL = 4;
    public const REJECTED = 5;
    public const EXPIRED = 6;
    public const EXPIREDINMATCH = 9;
    public const PENDINGNEW = 11;
    public const UNKNOWN = 253;
    public const NONREPRESENTABLE = 254;
    public const NULL_VALUE = 255;

    public static function encodedLength(): int
    {
        return 1;
    }
}

File: sbe/binance/spot_3_1/generated-python/spot_sbe/OrderStatus.py
"""Generated SBE (Simple Binary Encoding) message codec."""

import struct
from typing import List, Optional, Tuple, Set
from io import BytesIO

class OrderStatus:
    """Enum values."""
    NEW = 0
    PARTIALLY_FILLED = 1
    FILLED = 2
    CANCELED = 3
    PENDING_CANCEL = 4
    REJECTED = 5
    EXPIRED = 6
    EXPIRED_IN_MATCH = 9
    PENDING_NEW = 11
    UNKNOWN = 253
    NON_REPRESENTABLE = 254
    NULL_VALUE = 255

    @staticmethod
    def encoded_length() -> int:
        return 1
